x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/content.tsx

```diff
@@ -52,7 +58,10 @@ export function Content(props: ContentProps) {
 
 type ContentPanelProps = PackageInfo & Pick<DetailParams, 'panel'>;
```

Comment:
what do you think of changing this type to `{ packageInfo: PackageInfo, panel: DetailViewPanelName}` this seems weird to me as a prop type

Reply:
Yeah, I agree with that suggestion @nchaulet . Will refactor it